code/modules/client/patron.dm

```diff
@@ -25,11 +25,9 @@
 		var/key = ckey(line_list[1])
 
 		var/date
-		log_world("Doing key [key], date is [date]")
 		if (line_list.len > 1)
 			date = sanitize_date(line_list[2])	//This will return FALSE if the date does not look right
 		if (!date)
-			log_world("ERROR: Bad date found in patrons.txt for ckey [key], replacing this with a 1 month timer from now as a fallback")
```

Comment:
this one is fine to keep in live code, it actually indicates something bad which shouldn't happen